src/capture_RPiHQ.cpp

```diff
@@ -548,83 +522,70 @@ time ( NULL );
 	{
 		ss.str("");
 		ss << asiBrightness;
-		brightness = "--brightness " + ss.str() + " ";
+		command += " --brightness " + ss.str();
 	}
 
-	// Add brightness info to raspistill command string
-	command += brightness;
-
-	// Quality
-	string squality;
-
 	if (quality < 0)
 	{
 		quality = 0;
 	}
-
-	if (quality > 100)
+	else if (quality > 100)
 	{
 		quality = 100;
 	}
 
 	ss.str("");
 	ss << quality;
-	squality = "--quality " + ss.str() + " ";
-
-	// Add image quality info to raspistill command string
-	command += squality;
+	command += " --quality " + ss.str();
 
 	if (!darkframe) {
 		if (showDetails)
-			command += "-a 1104 ";
+			command += " -a 1104";
 
 		if (time==1)
-			command += "-a 1036 ";
+			command += " -a 1036";
 
 		if (strcmp(ImgText, "") != 0) {
 			ss.str("");
 			ss << ImgText; 
-			if (myModeMeanSetting.debugLevel > 0) {
+			if (debugLevel > 0) {
 				ss << " shutter:" << myRaspistillSetting.shutter_us 
 				<< " gain:"	<< myRaspistillSetting.analoggain;
-				if (myModeMeanSetting.debugLevel  > 1 ) {
+				if (debugLevel  > 1 ) {
```

Comment:
Also test code - maybe disturbing and incomprehensible

Reply:
LOL - it's definitely incomprehensible to anyone but you!!  But if it helps you tune the algorithm or debug problems, leave it, but possible increase the debug level.  I have a lot of debug statements in the ZWO code that's been very useful in improving its algorithm.
I don't think "myModeMeaSettings.debugLevel" is needed anymore - I was able to change off of them to either "debugLevel", or use the new Log() function to write something to the log file.  I believe the only place where "myModeMeaSettings.debugLevel" still exists is in the argv[] checks.